src/main/java/software/sigma/internship/config/SecurityConfig.java

```diff
@@ -22,6 +22,7 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                 .antMatchers("/").permitAll()
                 .antMatchers("/locale").permitAll()
                 .antMatchers("/statistic-data/latest").permitAll()
+                .antMatchers("/registration").permitAll()
```

Comment:
In other PR Yevhen refactored permitAll a little. After he will merge his PR make it in the same style